pkg/login/login.go

```diff
@@ -92,8 +92,7 @@ func (c *Client) challengeAuth() (string, error) {
 	resp, err := c.sniffAuth("")
 	if err != nil {
 		return "", err
-	}
-	if resp.StatusCode == 200 {
+	} else if resp.StatusCode == 200 {
```

Comment:
Do you find it more readable like this? I meant it the other way around, or even using a switch statement. See here for example how it's done to conditionally return based on the status code : https://github.com/dcos/dcos-core-cli/blob/1cbe3e815a54a7e1d43d8236690c504ab9765c55/pkg/metronome/client.go#L80

But anyways, that's fine as is too, not a big deal.